examples/docs-wisdom-yargs/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM reachsh/runner:0.1.10
+
+# If your project needs more Node dependencies, uncomment these next three
+# lines by removing the leading `#` character:
+
+ COPY package.json /app/package.json
+ RUN npm install
+ RUN npm link @reach-sh/stdlib
```

Comment:
I think I might be missing something. These three lines are already uncommented.

Reply:
I need to remove this file and a few others.